build.ps1

```diff
@@ -2,7 +2,7 @@ $scriptRoot = Split-Path $MyInvocation.MyCommand.Path -Parent
 $dotnetScriptPath = "$scriptRoot\build\dotnet-script"
 if (!(Test-Path $dotnetScriptPath)) {
     Write-Host "Downloading dotnet-script..."
-    Invoke-Expression ".\install-dotnet-script.ps1"
+    Invoke-Expression "$scriptRoot\build\install-dotnet-script.ps1"
```

Comment:
how did it work before?! I swear it did 😊

Reply:
`.\dotnet-script\build.ps1` did not work. On failes if the dotnet-script folder does not exists within the build folder.  :)

Reply:
god help me